src/java/org/apache/cassandra/cql3/UntypedResultSet.java

```diff
@@ -432,6 +432,15 @@ public long getLong(String column)
             return LongType.instance.compose(data.get(column));
         }
 
+        // this function will return the default value if the row doesn't have that column or the column data is null
+        // This function is used to avoid the nullpointerexception
+        public long getLongOrDefault(String column, long defaultValue) {
+            if (!has(column)){
+                return defaultValue;
+            }
+            return getLong(column);
+        }
```

Comment:
Nit: there are some existing methods (`getInt`, `getUUID`) that are overloaded with an additional `ifNull` parameter, for consistency we should do the same for getLong.

```suggestion
        public long getLong(String column, long ifNull) {
            ByteBuffer bytes = data.get(column);
            return bytes == null ? ifNull : LongType.instance.compose(bytes);
        }
```

Reply:
> Nit: there are some existing methods (`getInt`, `getUUID`) that are overloaded with an additional `ifNull` parameter, for consistency we should do the same for getLong.

ACK - will update it

Reply:
Done